### PR TITLE
feature locking: fix custom uno command handling of menu items

### PR DESCRIPTION
**Pull request by lpranam**
adding 'uno' field to menu items broke some custom feature handling
removed '.uno:' from the command and if uno field is missing add lockUno field

it is a revert of https://github.com/lpranam/online/commit/a6cc06ae88980bf039ca5a7417f53899eda412f1 with some additional changes

Signed-off-by: Pranam Lashkari <lpranam@collabora.com>
Change-Id: Ie6a1201bbff5c383458c1a4c03d4989189b14e3a

* Target version: master 

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Review by kendy — CHANGES_REQUESTED** [review]
Please let's not duplicate everything; it is quite easy to go though the original patch a6cc06ae88980bf039ca5a7417f53899eda412f1 and see where the 'uno' has been added - let's just change those uno:'s to lockUno's, to have this patch minimal & avoid lots of duplication.

**Comment by lpranam** [comment]
@kendy that's what patch does...

File: browser/src/control/Control.TopToolbar.js [hunk 1]
```diff
@@ -137,48 +137,48 @@ L.Control.TopToolbar = L.Control.extend({
 				}, mobile: false},
 			{type: 'break', id: 'breakfontsizes', invisible: true, mobile: false, tablet: false,},
 			{type: 'button', id: 'languagecode', desktop: false, mobile: true, tablet: false},
-			{type: 'button',  id: 'bold',  img: 'bold', hint: _UNO('.uno:Bold'), uno: '.uno:Bold'},
-			{type: 'button',  id: 'italic', img: 'italic', hint: _UNO('.uno:Italic'), uno: '.uno:Italic'},
-			{type: 'button',  id: 'underline',  img: 'underline', hint: _UNO('.uno:Underline'), uno: '.uno:Underline'},
-			{type: 'button',  id: 'strikeout', img: 'strikeout', hint: _UNO('.uno:Strikeout'), uno: '.uno:Strikeout'},
+			{type: 'button', id: 'bold', img: 'bold', hint: _UNO('.uno:Bold'), uno: 'Bold', lockUno: '.uno:Bold'},
```
Comment (kendy) [inline browser/src/control/Control.TopToolbar.js:140]: Cases like this are those I'm not happy about - we end up with both uno: 'Bold' and lockUno: '.uno:Bold' ; please let's avoid this :-)

File: browser/src/control/Control.TopToolbar.js [hunk 2]
```diff
@@ -124,7 +124,7 @@ L.Control.TopToolbar = L.Control.extend({
 						}});
 					}
 				}, mobile: false},
-			{type: 'html',   id: 'fontsizes',
```
Comment (kendy) [inline browser/src/control/Control.TopToolbar.js:127]: Also let's avoid whitespace changes please...

**Review by kendy — CHANGES_REQUESTED** [review]
I've clarified my concerns - hope it makes more sense now?  Thank you!

**Review by kendy — APPROVED** [review]
Great - thank you for all the updates, looks good to me!

**Pull request merged**